Composer/packages/server/src/models/bot/builder.ts

```diff
@@ -259,22 +260,28 @@ export class Builder {
    *
    * Models are placed as a sibling to ComposerDialogs by default
    */
-  public async copyModelPathToBot() {
+  public async copyModelPathToBot(featureFlags: FeatureFlagMap) {
     for (const lang in this.orchestratorSettings.orchestrator.models) {
       const modelName = Path.basename(this.orchestratorSettings.orchestrator.models[lang], '.onnx');
-      const destDir = Path.resolve(this.botDir, '..', MODEL, modelName);
+
+      const destDir = featureFlags.NEW_CREATION_FLOW.enabled
+        ? Path.resolve(this.botDir, MODEL, modelName)
+        : Path.resolve(this.botDir, '..', MODEL, modelName);
+
       await copy(this.orchestratorSettings.orchestrator.models[lang], destDir);
     }
 
-    await this.updateOrchestratorSetting();
+    await this.updateOrchestratorSetting(featureFlags);
   }
 
   /**
    * Orchestrator: Update Orchestrator Settings for publishing
    *
-   * Orchestrator file paths are updated relative to the project root.
+   * Models are located in <project root>/model
+   * In the Adaptive Runtime, Orchestrator snapshot files are located in <project root>/generated.
```

Comment:
Is this the correct assumption @benbrown

Reply:
Yes